myriad-scheduler/src/main/java/com/ebay/myriad/Main.java

```diff
@@ -190,6 +196,29 @@ private void startNMInstances(Injector injector) {
       }
     }
 
+    /**
+     * Create NMProfile for any configured service
+     * @param injector
+     */
+    private void initJavaBasedTaskConfiguration(Injector injector) {
+      LOGGER.info("Initializing JavaBasedTaskConfiguration");
+      NMProfileManager profileManager = injector.getInstance(NMProfileManager.class);
+
+      Map<String, AuxTaskConfiguration> auxServicesConfigs = 
+          injector.getInstance(MyriadConfiguration.class).getAuxTaskConfigurations();
+      if (auxServicesConfigs != null) {
+        for (Map.Entry<String, AuxTaskConfiguration> entry : auxServicesConfigs.entrySet()) {
+          AuxTaskConfiguration config = entry.getValue();
+          Double cpu = (config.getCpus().isPresent() ? config.getCpus().get() : null);
+          Double mem = (config.getJvmMaxMemoryMB().isPresent() ? config.getJvmMaxMemoryMB().get() : null);
+          
+          if (cpu != null && mem != null) {
+            profileManager.add(new NMProfile(entry.getKey(), cpu, mem));
+          }
```

Comment:
I am going to use default values Marathon is using: 0.1 CPU, 16 MB memory instead of nulls